src/qcodes_contrib_drivers/drivers/QuTech/M2q.py

```
@@ -0,0 +1,176 @@
+    def get_raw(self) -> tuple[ParamRawDataType, ...]:
+        """
+        Get raw values from the M2q preamplifier.
+
+        Returns:
+            Tuple of (voltage_raw, voltage) where voltage is calculated
+            from voltage_raw using the total gain.
+        """
+        if not isinstance(self.instrument, M2q):
+            raise TypeError(
+                f"Expected instrument to be M2q, got {type(self.instrument).__name__}"
+            )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)